### ISSUE TITLE
Some project settings are missing from the ProjectSettings documentation

### ISSUE THREAD

**Issue by akien-mga** [godotengine/godot#55730]
### Godot version

4.0.dev (e53e357), also 3.x

### System information

Linux, Mageia 9 x86_64

### Issue description

Some project settings (defined throughout the codebase with `GLOBAL_DEF` or derivatives) seem to be missing from the list of properties in `doc/classes/ProjectSettings.xml` (and thus its in-editor and online versions: https://docs.godotengine.org/en/latest/classes/class_projectsettings.html).

This typically happens when the codepath where the project setting is defined is not triggered by running `--doctool` to generate the class reference: https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html

Here's a list of the project settings I found defined in the codebase yet missing in the docs for the `master` branch:

```
application/config/auto_accept_quit
application/config/quit_on_go_back
application/run/main_loop_type
collada/use_ambient
debug/disable_touch
debug/shader_language/warnings/enable
debug/shader_language/warnings/treat_warnings_as_errors
debug/shader_language/warnings/ + ShaderWarning::get_name_from_code(code).to_lower()   // more than one warning
display.iOS/use_cadisplaylink
display/window/per_pixel_transparency/allowed
display/window/per_pixel_transparency/enabled
display/window/stretch/aspect
display/window/stretch/mode
display/window/stretch/shrink
display/window/subwindows/embed_subwindows
editor/export/convert_text_resources_to_binary
editor/import/reimport_missing_imported_files
editor/import/use_multiple_threads
gui/common/snap_controls_to_pixels
gui/fonts/dynamic_fonts/use_oversampling
physics/3d/active_soft_world
physics/3d/smooth_trimesh_collision
rendering/limits/buffers/canvas_polygon_buffer_size_kb
rendering/limits/buffers/canvas_polygon_index_buffer_size_kb
rendering/quality/2d/use_pixel_snap
rendering/textures/canvas_textures/default_texture_filter
rendering/textures/canvas_textures/default_texture_repeat
```

### Steps to reproduce

```
rg GLOBAL_DEF | sed -e 's/.*GLOBAL_DEF.*("\([^"]*\)".*/\1/g' | sort -u > global-defs.txt
grep "member name=" doc/classes/ProjectSettings.xml | sed -e "s/[^\"]*\"\([^\"]*\)\".*/\1/g" > global-docs.txt
```
Cleanup extracted strings which are not what you want and diff the two files.

**Comment by dalexeev** [comment]
`editor/*` project settings:

* core/config/project_settings.cpp - ok
* editor/editor_file_system.cpp - missing
* editor/editor_node.cpp - missing
* editor/export/editor_export_plugin.cpp - missing
* editor/plugins/version_control_editor_plugin.cpp - missing
* scene/main/node.cpp - ok
* servers/movie_writer/movie_writer.cpp - ok


See also: godotengine/godot-proposals#4771

**Comment by KoBeWi** [comment]
`rg GLOBAL_DEF | sed -e 's/.*GLOBAL_DEF.*("\([^"]*\)".*/\1/g' | sort -u > global-defs.txt` does not work anymore after #68429.

**Comment by akien-mga** [comment]
Fixed by #80972.
